File: 18) Defanging an IP Address/code.py
def defangIPaddr(address):
    defangedIP = ''
    for c in address:
        if c == '.':
            defangedIP += '[.]'
        else:
            defangedIP += c
    return defangedIP

address = input("Enter IP address: ")
defangedAddress = defangIPaddr(address)
print(defangedAddress)

File: 18) Defanging an IP Address/code.java
import java.util.Scanner;

public class Main {
    public static String defangIPaddr(String address) {
        StringBuilder defangedIP = new StringBuilder();
        for (char c : address.toCharArray()) {
            if (c == '.') {
                defangedIP.append("[.]");
            } else {
                defangedIP.append(c);
            }
        }
        return defangedIP.toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter IP address: ");
        String address = scanner.nextLine();
        String defangedAddress = defangIPaddr(address);
        System.out.println(defangedAddress);
    }
}
